ui/apps/platform/src/Containers/Vulnerabilities/components/AdvancedFiltersToolbar.tsx

```diff
@@ -125,19 +127,22 @@ function AdvancedFiltersToolbar({
                         onSearch={onFilterApplied}
                     />
                 </ToolbarGroup>
-                {includeCveFilters && (
+                {(includeCveSeverityFilters ||
+                    (includeCveStatusFilters && isFixabilityFiltersEnabled)) && (
```

Comment:
Do to the above prop being split, we need to do an additional check here so that the `<ToolbarGroup />` is not rendered when both props are false.